lib/apiservers/portlayer/restapi/handlers/storage_handlers.go

```diff
@@ -237,3 +281,16 @@ func convertImage(image *spl.Image) *models.Image {
 		Store:    image.Store.String(),
 	}
 }
+
+//utility functions
+
+func volumeToCreateResponse(volume *spl.Volume, model *models.VolumeRequest) models.VolumeResponse {
+	response := models.VolumeResponse{
+		Driver:   model.Driver,
+		Name:     volume.ID,
+		SelfLink: volume.SelfLink.String(), //FIXME: is this right?
```

Comment:
This is a very scary FIXME right there. Why do we have it, what's not clear and what can we do to clarify?